CHANGES_NEXT_RELEASE

```diff
@@ -16,3 +16,4 @@
 - Fix: first observable value is processed and forwarded to the CB (#73).
 - Add: allow NGSIv2 for updating active attributes at CB, through configuration (#104)
 - Using precise dependencies (~=) in packages.json
+- Add: supports NGSIv2 for device provisioning (entity creation and context registration) at CB (#233)
```

Comment:
It seems that #233 doesn't exists as an issue in this repo...

Reply:
Fixed with 0196c4629d6b73640d7bd26a0b183ecc80e2f677